app/views/coronavirus_landing_page/components/landing_page/_page_header.html.erb

```diff
@@ -24,30 +24,40 @@
         <div class="govuk-grid-column-two-thirds">
           <%= render "govuk_publishing_components/components/heading", {
             text: sanitize(header["title"]),
-            font_size: 19,
+            font_size: 24,
             margin_bottom: 6,
             inverse: true
           } %>
-          <ul class="covid__instruction-list">
-            <% icons = [
-              "icon-hand-wash",
-              "icon-face-cover",
-              "icon-make-space"
-            ] %>
-            <% header["list"].each_with_index do | item, index | %>
-              <li class="covid__instruction-list-item">
-                <% if item.is_a?(Hash) %>
-                  <%= render "coronavirus_landing_page/components/landing_page/#{icons[index]}.svg" if icons[index] %>
-                  <span class="covid__instruction-text">
-                    <strong class="covid__instruction"><%= item["instruction"].capitalize %></strong>
-                    <%= item["detail"] %>
-                  </span>
-                <% else %>
-                  <%= item %>
-                <% end %>
-              </li>
+
+          <% if header["description"] %>
+            <%= render "govuk_publishing_components/components/govspeak", {
+            } do %>
+              <%= sanitize(header["description"]) %>
```

Comment:
Do you mean to use sanitize here? Is this used somewhere else with HTML?

Reply:
I think sanitizing everything that comes from a yaml is good practice – trust but verify.

Reply:
Hmm I think this seems a bit fishy. The string will already have html entities stripped because it's not marked as html_safe.

However when we're using the govspeak component we actually expect the content to be markdown that is converted to HTML whereas this was just be a string of content so there'd be no govspeak elements to style?

Reply:
As far as I know, the govspeak component is yielding a block, it doesn't know or care what that is and we can write anything in that yaml file in `coronavirus_landing_page` – hence I felt that sanitizing that might not be a bad idea. Otherwise, someone can add a script tag successfully and we'll enjoy the show.

Reply:
I pinged you on Slack as I'm not sure we're on the same wavelength here but given Slack's problems today I'll try explain in a comment too.

So first concern is the info going into this isn't Govspeak HTML.

For example if `header["description"] = "This is some text to show"`

Then the output will be:

```
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">This is some text to show</div>
```

Note that is not any paragraph elements like we'd expect.

---

Less of a concern is the sanitize front. If `header["description"] = "This is some malicious html <script>alert</script>"`

Then the output without calling the sanitize function would be:

```
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
              This is some malicious html &lt;script&gt;alert&lt;/script&gt;

</div>
```

because we've never run html_safe on `header["description"]` it'll have all HTML escaped by default.

Reply:
Not sure where at which point the HTML is escaped from the yml to the output, but if that's the case, happy to drop the sanitize.

Reply:
Just tested now if I'm not using `sanitize` then the following

```
<% header["description"] = "<p>Some content</p>" %> 
<%= render "govuk_publishing_components/components/govspeak", { } do %>
  <%= sanitize(header["description"]) %>
<% end %>
```

 will result in `<p>Some content</p>` instead of `Some content`

Reply:
Note my comment in the PR description that by design, we are required to have a few <p> tags in the yaml

Reply:
Clarified this offline and changed the approach to use locally defined content and `render_govspeak`.